src/Application/Migrations/23_function_encoding_profile_all_dependee.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION encoding_profile_all_dependees(param_profile_id bigint, param_known_ids bigint[] DEFAULT array[]::bigint[])
+  RETURNS TABLE (id bigint) AS
+$$
+DECLARE
+  p bigint;
+  ids bigint[];
+BEGIN
+  IF array_dims(param_known_ids) ISNULL THEN
+    param_known_ids := array[param_profile_id];
+  END IF;
+
+  FOR p IN
+    SELECT p.id
+    FROM tbl_encoding_profile p
+    WHERE depends_on=param_profile_id AND NOT (p.id = ANY(param_known_ids))
+  LOOP
+    ids := p || ids;
+    ids := (SELECT ARRAY(SELECT e.id from encoding_profile_all_dependees(p, ids || param_known_ids) e)) || ids;
```

Comment:
Mh, I thought this would creates duplicates when returning from recursion, but with an example I tried on paper I coulnd't reproduce this, so I guess this is just correct.

Reply:
Does it have to have that outer SELECT? Does ids := ARRAY() || ids throw errors?